### ISSUE TITLE
NC13b3 regression: 'Name' from Theming should not be used in 'Version'

### ISSUE THREAD

**Issue by nursoda** [nextcloud/server#7578]
### Steps to reproduce
1. Upgrade to NC 13b3
2. Set a 'Name' in Theming, e.g. NAME
3. Open https://seyfarth.de/cloud/index.php/settings/admin

### Expected behaviour
Version should read "Nextcloud 13.0.0 beta 3"
### Actual behaviour
Version reads "NAME 13.0.0 beta 3"

Note that this also baffled the author of the talk app since he used it (until a couple of days ago) to identify that the server is a nextcloud instance. 'Name' should be a display-only value, not to identify the product. The product neets a separate variable / identifier (and probably has). This should be used to display the "Product + Version" here, too.

**Comment by MorrisJobke** [comment]
That's not really a regression, because we did this in the past always like that.

https://github.com/nextcloud/server/blob/master/settings/templates/settings/admin/server.php#L226

@karlitschek What is your opinion here? I would be fine to hardcode "Nextcloud" there, because it is also only visible on the admin settings anyways.

cc @rullzer @blizzz

**Comment by rullzer** [comment]
The thing @mario ran into was in the status.php I bet. So it is not exactly the same issue.

I don't have a strong preference here. But I guess some buisnesses want the branded name in status.php.

**Comment by mario** [comment]
Yes, I used productname in status.php to identify the instance as Nextcloud. I have since removed the check since it's easily change-able anyway :)

**Comment by karlitschek** [comment]
There are scenarios it should be possible to hide 'Nextcloud' completely. Even for the Admin. This is mainly for 'real' themes of course. So not sure we should change the current behaviour.

**Comment by nursoda** [comment]
I'm fine with "MyName Version", but maybe you should just omit NAME in settings/general in the "Version" section?
